storage/StorageManager.cpp

```diff
@@ -487,23 +481,18 @@ MutableBlockReference StorageManager::getBlockInternal(
     const block_id block,
```

Comment:
Most changes here was a part of reverting a merged PR. We could add comments in a separate PR.